core/include/moveit/task_constructor/solvers/planner_interface.h

```diff
@@ -90,6 +90,9 @@ class PlannerInterface
 	                  const moveit::core::JointModelGroup* jmg, double timeout,
 	                  robot_trajectory::RobotTrajectoryPtr& result,
 	                  const moveit_msgs::msg::Constraints& path_constraints = moveit_msgs::msg::Constraints()) = 0;
+
+	// get name of the planner
+	virtual std::string getPlannerId() const = 0;
```

Comment:
Instead of pure virtual, should we default this to a blank string or something so we keep backward compatibility?

Reply:
I'd like this to make a breaking change to have the planner_info field consistently populated and it is very easy to update in any existing planner.